hack/BUILD.bazel

```diff
@@ -183,6 +185,17 @@ sh_test(
     ],
 )
 
+sh_binary(
```

Comment:
We need a call to test and also a call to fix the problems. I am only seeing one new target. So when we run bazel test //hack/... We run a test.

Reply:
Correct me if I understood it wrong, you are saying to add a target to fix the lint problems something like `gofmt -w`.
But golangci-lint issues can't be solved directly, those needs to be fixed manually by the developer.
Also, if we add a test right now, it's going to fail as there are many golangci-lint in master at the moment

Reply:
@chrislovecnm Thoughts?

Reply:
I _think_ what he's referring to is to have the ability to run `bazel test //hack/...` and have the linter run.

```bazel
sh_test(
  name = "verify-golangci-lint",
  srcs = ["//hack/bin:golangci-lint"],
  data = ["@//:all-srcs"],
)
```

If we do that, we don't need a separate GH action, since `make test/all` (which CI runs) will include the tests from `//hack`